include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -98,12 +98,18 @@ class UTVariadicRTLOp<string mnemonic, list<OpTrait> traits = []> :
 
 // Arithmetic and Logical Binary Operations.
 def AddOp : UTVariadicRTLOp<"add", [Commutative]>;
+def AddSignedOp : UTVariadicRTLOp<"add.s", [Commutative]>;
 def SubOp : UTBinRTLOp<"sub">;
+def SubSignedOp : UTBinRTLOp<"sub.s">;
 def MulOp : UTVariadicRTLOp<"mul", [Commutative]>;
+def MulSignedOp : UTVariadicRTLOp<"mul.s", [Commutative]>;
```

Comment:
I don't think we need these for add/sub/mul: given that the input and output widths are the same, the signed and unsigned versions of these do the same computation.